utilities/codeship_restart_build.sh

```diff
@@ -0,0 +1,97 @@
+#!/home/rof/.rvm/rubies/ruby-2.2.5/bin/ruby
```

Comment:
I would replace that with `#!/usr/bin/env ruby`, that way that script works outside our build VMs as well.